packages/mdc-drawer/README.md

```diff
@@ -121,9 +127,15 @@ Drawers can contain a header element which will not scroll with the rest of the
   </div>
   <div class="mdc-drawer__content">
     <nav class="mdc-list">
-      <a class="mdc-list-item mdc-list-item--activated" href='#'>
+      <a class="mdc-list-item mdc-list-item--selected" href='#' tabindex="0">
```

Comment:
I noticed you reverted `selected` to `activated` where I commented, but there are still a bunch of changes from `activated` to `selected` in this PR.